README.md

```diff
@@ -29,10 +33,47 @@ bot will try to release on Fedora dist-git, on `master` branch and branches spec
 It should not create merge conflicts, but in case it does,
 you have to solve them first before attempting the release again.
 
+# Try it locally
+For now, the easiest way how to install release-bot is via pip.
+Other possible installations are through
+[Docker](#docker-image), [OpenShift](#openshift-template).
+```
+$ pip install release-bot
+``` 
+First interaction with release bot may be automated releases on Github. Let's do it. 
+
+#### 1. Create upstream repository 
+First of all, create a Github repository or use an existing one. This will be upstream repository
+were releases will be published. Within upstream repository create `release-conf.yaml` file which contains info on how to release the specific project.
+
+Copy and edit [release-conf.yaml](release-conf-example.yaml). For possible advanced setup check [the documentation for an upstream repository](#upstream-repository).
+
+#### 2. Create another private repository
+In the next step, create another private Github or local repository for the purpose of storing config files with access tokens.
 
-# Configuration
+Within private repository copy and edit [conf.yaml](conf.yaml) to allow automatic Github releases. You will need to generate a [Github personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/).
+Recommended permissions for access token are: `repo`, `delete_repo`, `user`. Later on, this repository will store also `.pypirc` file for PyPi configuration.
+For possible advanced setup check [the documentation for a private repository](#private-repository).
+
+#### 3. Run release-bot
+At this point, release-bot is installed. At least two configuration files are set `release-conf.yaml` and `conf.yaml` (optionally `.pypirc`).
+ 
+ Launch bot by a command:  
+```$ release-bot -c <path_to_conf.yaml> --debug```  
+You can scroll down and see debug information of running bot.
+
+#### 4. Make a new release
+- Create a PR named `0.0.1 release` in your upstream repository. You can select your own version number.
```

Comment:
```suggestion
- Create an issue having `0.0.1 release` as a title in your upstream repository. You can select your own version numbers.
```

Reply:
I guess this is correct. If you are using default configuration you have to create PR, because `trigger_on_issue` option is False by default.

Reply:
Thank you for the review. I agree with all the suggested changes above. But @rpitonak is right, also `github_username` must be added in `conf.yml`. I can change this guide for releases based on issues but default configuration files have to be changed.

Reply:
+1 to change it so that people utilize trigger_on_issue=True; I always forget it's off by default